MatrixSDK/MXRestClient.h

```diff
@@ -1444,7 +1444,7 @@ NS_REFINED_FOR_SWIFT;
  @return a MXHTTPOperation instance.
  */
 - (MXHTTPOperation*)stateOfRoom:(NSString*)roomId
-                        success:(void (^)(NSDictionary *JSONData))success
+                        success:(void (^)(NSArray *JSONData))success
```

Comment:
This type was wrong. AFNetworking will return responseObject of type `id`, so we cannot generally assume it is always a dictionary. In this particular case the API actually returns an array. We never had issues with this, because this API was not being used by the app until now.